travis/setup-guest

```diff
@@ -13,7 +13,7 @@ while check_network ; [ $? -ne 0 ];do
 done
 
 apt-get -qq update
-apt-get install -y qemu libvirt-bin libvirt-dev golang-1.8 ovmf
+apt-get install -y qemu-system-common libvirt-bin libvirt-dev golang-1.8 ovmf
```

Comment:
This is covered in one of your other PRs already.

Reply:
Yes, I can pull the other PR or remove from here, but travis fails without this.

Reply:
@monstermunchkin shall I remove from this PR or close the other ?

Reply:
I've closed the other PR, but please also remove this commit from this PR since it's not needed.